### PR TITLE
Initial Jenkins jobs

### PR DESCRIPTION
**Pull request by kszarlej**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+podTemplate(label: label, yaml: readTrusted('JenkinsPods.yaml')) {
+  node(label) {
+
+    ci {
+      gitCheckout()
+
+      stage('download Go deps') {
+        container('golang'){
+          sh 'apk add --no-cache git gcc libc-dev'
```
Comment (kszarek) [inline Jenkinsfile]: why do you install gcc and libc since CGO is disabled https://github.com/AirHelp/filler/blob/master/Dockerfile-test#L3??

**Comment by kszarlej** [comment]
@kszarek it used vanilla golang image which had cgo enabled by default. I disabled it using env vars

File: JenkinsPods.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: golang
+spec:
+  containers:
+  - name: golang
+    image: golang:1.11.2-alpine
```
Comment (jazzbee) [inline JenkinsPods.yaml]: why not use golang:1.12-alpine?

Comment (kszarlej) [reply JenkinsPods.yaml]: I just used the one from here https://github.com/AirHelp/filler/blob/master/Dockerfile-test#L1

**Pull request merged**